zerver/openapi/zulip.yaml

```diff
@@ -9122,85 +9141,72 @@ paths:
                       presence_last_update_id:
                         type: integer
                         description: |
-                          Provides the `last_update_id` value of the latest presence data
-                          fetched by the server and included in the response in
-                          `presences`. This allows the client to have an identifier number
-                          up to which it knows it has obtained all the presence data.
+                          The identifier for the latest user presence data returned in
+                          the `presences` data of the response.
+
+                          If a value was passed for `last_update_id`, then this is
+                          guaranteed to be equal to or greater than that value. If it
+                          is the same value, then that indicates to the client that
+                          there were no updates to previously received user presence
+                          data.
+
+                          The client should then pass this value as the `last_update_id`
+                          parameter when it next queries this endpoint, in order to only
+                          receive new user presence data and avoid redundantly fetching
+                          already known information.
 
-                          The client should then pass this value when it next queries this
-                          endpoint, in order to only receive newer presence data to avoid
-                          redundant fetching of already known information.
+                          This will be `-1` if user presence is disabled for the
+                          organization, or no value was passed for
+                          [`last_update_id`](#parameter-last_update_id) and no user
+                          presence data was returned from the server.
```

Comment:
It felt important to have an explanation for when clients might see `-1` for this return value.

Reply:
Yeah, good point.

Reply:
`was returned from the server` feels more readable to me.